### DISCUSSION TITLE
Bulk Crap Uninstaller showing 2 Versions

### DISCUSSION THREAD

**Discussion by Bass303** [sumatrapdfreader/sumatrapdf#4002] [Q&A] [answered]
I seem to have 2 versions of Sumatra pdf, the latest 3.5.2 and 3.3.3.
When I open Sumatra pdf I have it set to show the last documents I was viewing.  Is it possible to remove 3.3.3 but will it still show the last documents I was viewing?
Thanks in advance.

**Comment by GitHubRulesOK** [comment, marked as answer]
you need to back-up your settings so in the version that you have all your customisation **enter**   `settings advanced options` and file **save AS** to a known place say your desktop as `SumatraPDF-settings.txt`. then you can safely delete either or both and know you have a current backup.
If you delete both and decide to install a fresh one you dont open a file simply start install SumatraPDF and note where the advanced settings are going to be. COPY the one from desktop to that new location and on first start it should clean out any bad settings and should keep the good ones.

**Reply by Bass303** [reply]
Worked perfectly.  Thanks again.
